public/js/app/components/CourseFacts.jsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react'
+import i18n from '../../../../i18n'
 
 import { linkToSchool } from '../util/links'
+import { getDateFormat, seasonStr } from '../util/helpers'
 
-const formatRounds = rounds => {
+const formatRoundsShort = (language, memoData) => {
   // Split rounds with comma after end parentheses and then add '),' in display
-  const splitRounds = rounds.split('),')
-  const lastIndex = splitRounds.length - 1
+  const langIndex = language === 'en' ? 0 : 1
+  const splitRounds = memoData.memoName.split('),')
   return (
     <>
-      {splitRounds.map((round, thisIndex) => (
-        <span key={round}>
-          {`${round}${thisIndex === lastIndex ? '' : ')'}`}
-          <br />
-        </span>
-      ))}
+      {splitRounds.map((round, thisIndex) => {
+        let shortName = round.slice(0, 7)
```

Comment:
Why it slices until index 7? Could add a comment or a descriptive name to a number 7

Reply:
traying to use more descriptive way to build short name